src/main/java/by/exadel/internship/service/impl/EmailServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package by.exadel.internship.service.impl;
+
+import by.exadel.internship.dto.formDTO.FormRegisterDTO;
+import by.exadel.internship.mail.EmailProperties;
+import by.exadel.internship.mail.EmailTemplate;
+import by.exadel.internship.service.EmailService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.Map;
+import java.util.Properties;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class EmailServiceImpl implements EmailService {
+
+    private final EmailProperties emailProperties;
+    private JavaMailSenderImpl mailSender;
+
+    private static final Map<String,String> MAIL_PROPS = Map.of("mail.transport.protocol", "smtp",
```

Comment:
Why properties is not declared in `application.yaml`?

Reply:
done